superset-frontend/src/features/alerts/AlertReportModal.tsx

```diff
@@ -593,6 +605,22 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
     setNotificationAddState('active');
   };
 
+  const updateAnchorState = (value: any) => {
```

Comment:
Can we use more specific type?

Reply:
Ideally, I'd like for this to be typed as  `string | undefined`, but I'm running into issues from antd's typing for the `onSelect` prop which is typed as `'RawValueType | LabelValueType'` and throws an error on compile when typed as such. I've used any as a workaround.